ui/src/submissions/conferences/components/ExistingConferencesAlert.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from 'antd';
+import FieldInfoAlert from '../../common/components/FieldInfoAlert';
+import pluralizeUnlessSingle from '../../../common/utils';
+import ExistingConferencesDrawer from './ExistingConferencesDrawer';
+
+function ExistingConferencesAlert({
+  onDatesChange,
+  dates,
+  numberOfConferences,
+}) {
+  const [openingDate, closingDate] = dates;
+  const [isDrawerVisible, setDrawerVisible] = useState(false);
+
+  const showDrawer = useCallback(visible => setDrawerVisible(visible), []);
+
+  useEffect(
+    () => {
+      if (openingDate && closingDate) {
+        onDatesChange(openingDate, closingDate);
```

Comment:
`onDatesChange([openingDate, closingDate])` to be consistent with how we receive them.